src/nytt/useAntallUlesteNyheter.ts

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+import { Nyhet } from './Nytt';
+
+const hentAntallUlesteNyheter = (nyheter: Nyhet[], antallLesteNyheter: number) => {
+    if (nyheter.length === 0) {
+        return 0;
+    }
+
+    return nyheter.length - antallLesteNyheter;
+};
+
+const LOCAL_STORAGE_KEY = 'antallLesteNyheter';
+
+const useAntallUlesteNyheter = (
+    nyheter: Nyhet[],
+    onFørsteBesøk: () => void
+): [number, number, () => void] => {
+    const [antallUlesteNyheter, setAntallUlesteNyheter] = useState<number>(0);
+    const [antallUlesteVedSidelast, setAntallUlesteVedSidelast] = useState<number>(0);
```

Comment:
Holder på to states nå. Antall uleste nyheter, og antall uleste du hadde da du lastet inn siden. Sistnevnte brukes kun for å fremheve nyhetene som var uleste da du kom inn på siden. For med en gang du åpner nyhetsboksen vil "antallUlesteNyheter" settes til (se "markerSomLest"-funksjonen lenger ned)